gamemode/modules/animations/sh_animations.lua

```diff
@@ -1,7 +1,7 @@
 local Anims = {}
 
 -- Load animations after the languages for translation purposes
-hook.Add("loadCustomDarkRPItems", "loadAnimations", function()
+hook.Add("Think", "loadAnimations", function()
```

Comment:
I'm not sure I understand the problem. Looking quickly at the rest of the file, it seems that the server only uses `Anims` to see whether an animation is registered. It ignores the actual translation. The client does show the animation in the menu.

Other than that, instead of creating and removing a think hook, on can run a `timer.Simple(0,...)`. Note that either option will run the code quite a bit later than now.

Reply:
Never mind, I'm not able to reproduce this issue. This may be due to my misconfiguration on my part. I'll revert this change.